tests/test_code_gen.py

```diff
@@ -476,6 +476,15 @@ def test_non_string_leakage(self):
         '''
         self.assertAstRoundtrips(source)
 
+    def test_fstring_trailing_newline(self):
+        """
+        Check that trailing newlines in f-strings are printed correctly
```

Comment:
Nit: We can now drop this docstring because the test name itself is pretty self-descriptive.

Reply:
Sure. Dropped.